### DISCUSSION TITLE
Problem with Mask Generation

### DISCUSSION THREAD

**Discussion by benvillasana** [BrainEnergyLab/Inflammation-Index#17] [Fiji Plugin Issues]
Hello,
An error has emerged from Mask Generation.  I'm trying to analyze microglial morphology from immunofluorescence images (no z stacks). The former steps ran well. In this step (mask generation), I used the default values (200, 800, 100, 100, 120), but the following error emerged:

'.' expected in line 524
(called from line 813)
(called from line 1310)

selectWindow (Image <)>; 

Could you help me please?

**Comment by DKClarke** [comment]
Hi Benjamin,

Thanks for getting in touch and apologies for the inconvenience. I noticed this same error myself (seems to be related to changes to ImageJ) and have a fix I aim to deploy by Monday. I’ll send you a message when it’s done. 

All the best,

Devin

**Comment by DKClarke** [comment]
Hi @benvillasana my apologies for the delay - I was having issues building the plugin on my new laptop but have now pushed the latest plugin version (1.6) to the repo. Please let me know if it has resolved your issues.
